---2017-5-10 14:1:53【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:1:53【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:2:31【Error: SQLITE_MISUSE: Database is closed
    at Error (native)
    at Database.object.(anonymous function) [as close] (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\sqlite3\lib\trace.js:31:20)
    at Object.sqlite.close (F:\apiserver-191升级包5.5\apiserver-191升级包\func\sqlite.js:51:14)
    at Object.exports.index (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\admin_control.js:101:9)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\smrz.js:5:14)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:2:59【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:56:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:3:0【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:56:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:3:22【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:56:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:5:0【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:56:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:5:22【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:56:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:7:43【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:7:43【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:11:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:11:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:12:34【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:12:34【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:13【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:50:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:13:14【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:13:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:14:5【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:15:0【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:50:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:15:1【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:50:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:15:6【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers.js:50:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:16:43【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_apply.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:17:5【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_apply.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:18:43【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_apply.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:19:5【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_apply.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:21:5【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_apply.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:23:10【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_apply.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:25:40【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_apply.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:28:40【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_apply.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 14:31:40【error: terminating connection due to administrator command
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:40【error: terminating connection due to administrator command
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:40【error: terminating connection due to administrator command
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:40【error: terminating connection due to administrator command
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:40【error: terminating connection due to administrator command
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:40【error: terminating connection due to administrator command
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:41【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:42【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:42【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:42【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:42【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:43【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:49【error: the database system is shutting down
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:49【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:31:49【error: the database system is shutting down
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:49【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:31:49【error: the database system is shutting down
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:49【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:31:50【error: the database system is shutting down
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 14:31:50【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 14:31:50【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 14:45:3【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 14:45:3【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:15:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:31:29【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:34:48【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:35:6【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:35:6【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:35:6【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:35:6【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:35:6【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:35:6【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:36:42【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_users.js:42:8)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 15:36:55【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\servers_users.js:42:8)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 15:39:2【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:39:2【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:39:2【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【错误:连接PG数据库失败,错误信息:Error: Connection terminated】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:45:26【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:45:26【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:46:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:48:25【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:39【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:39【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:50:40【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:51:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:51:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:52:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:53:0【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:54:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:54:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:54:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:54:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:55:17【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\payment_order.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 15:55:46【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\payment_order.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 15:55:59【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:55:59【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:56:44【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:56:44【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:57:17【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\payment_order.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 15:57:23【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:57:23【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:57:27【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:57:27【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:57:27【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:57:27【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:57:27【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:57:27【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:57:27【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:57:27【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:57:27【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:57:27【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:57:38【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\payment_order.js:37:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:8【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:9【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:9【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:9【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:9【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:9【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:9【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:9【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:9【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:58:15【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:58:15【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 15:59:59【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 15:59:59【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 15:59:59【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:2:14【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:2:14【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【错误:连接PG数据库失败,错误信息:Error: Connection terminated】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:4【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【错误:连接PG数据库失败,错误信息:Error: Connection terminated】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:3:51【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:46【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:53【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:53【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:4:59【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:4:59【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:7:17【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:21【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: connect ETIMEDOUT 192.168.0.100:5432
    at Object.exports._errnoException (util.js:870:11)
    at exports._exceptionWithHostPort (util.js:893:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1061:14)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:8:49【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:11:26【错误:连接PG数据库失败,错误信息:Error: connect ETIMEDOUT 192.168.0.100:5432】
---2017-5-10 16:13:1【error: the database system is starting up
    at Connection.parseE (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:539:11)
    at Connection.parseMessage (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:366:17)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:531:20)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: read ECONNRESET
    at exports._errnoException (util.js:870:11)
    at TCP.onread (net.js:552:26)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 16:13:1【Error: Connection terminated
    at null.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\client.js:183:17)
    at g (events.js:260:16)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at Socket.<anonymous> (F:\apiserver-191升级包5.5\apiserver-191升级包\node_modules\pg\lib\connection.js:66:10)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:477:12)】
---2017-5-10 17:34:32【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\exchange.js:24:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 17:36:32【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\exchange.js:24:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
---2017-5-10 17:38:32【TypeError: Cannot read property '0' of undefined
    at Object.module.exports.lists (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\admin\share.js:76:41)
    at Object.module.exports.run (F:\apiserver-191升级包5.5\apiserver-191升级包\routes\Tpl\adminfunc\exchange.js:24:12)
    at F:\apiserver-191升级包5.5\apiserver-191升级包\routes\app_routes_main.js:53:20】
